src/teleop/gui/src/components/AutonWaypointItem.vue

```diff
@@ -94,12 +101,8 @@ export default {
 </script>
 
 <style scoped>
-.waypoint-item {
-  background-color: rgb(180, 180, 180);
-  border-radius: 5px;
+.box {
```

Comment:
Would be good to call it waypoint-item-wrap or something of the like. box isn't very descriptive.